pkg/client/cli/cmd_status.go

```diff
@@ -69,18 +69,35 @@ func statusCommand() *cobra.Command {
 		Use:  "status",
 		Args: cobra.NoArgs,
 
-		Short: "Show connectivity status",
-		RunE:  s.status,
+		Short:             "Show connectivity status",
+		RunE:              s.status,
+		PersistentPreRunE: fixFlag,
 		Annotations: map[string]string{
 			ann.RootDaemon: ann.Optional,
 			ann.UserDaemon: ann.Optional,
 		},
 	}
 	flags := cmd.Flags()
 	flags.BoolVarP(&s.json, "json", "j", false, "output as json object")
```

Comment:
Can we remove this flag? For neatness, I would prefer that we have one json flag that also influences the behavior of certain subcommands like list and status

Reply:
I would love to remove it, but removing it means breaking backward compatibility. So I hide it instead, and let it have the exact same meaning as `--output=json`.